src/test/java/com/ixortalk/organization/api/AbstractSpringIntegrationTest.java

```diff
@@ -296,8 +268,8 @@ public void jwtSetUp() {
         when(jwtDecoder.decode(ADMIN_JWT_TOKEN)).thenReturn(buildJwtTokenWithEmailCustomClaim(ADMIN_JWT_TOKEN, empty(), "admin", "ROLE_ADMIN"));
         when(jwtDecoder.decode(USER_JWT_TOKEN)).thenReturn(buildJwtTokenWithEmailCustomClaim(USER_JWT_TOKEN, of(USER_EMAIL), "user", "ROLE_USER"));
         when(jwtDecoder.decode(OTHER_USER_JWT_TOKEN)).thenReturn(buildJwtTokenWithEmailCustomClaim(OTHER_USER_JWT_TOKEN, of(OTHER_USER_EMAIL), "user", "ROLE_USER"));
-        when(jwtDecoder.decode(USER_IN_ORGANIZATION_X_ADMIN_ROLE_JWT_TOKEN)).thenReturn(buildJwtTokenWithEmailCustomClaim(USER_IN_ORGANIZATION_X_ADMIN_ROLE_JWT_TOKEN, of(USER_IN_ORGANIZATION_X_ADMIN_ROLE_EMAIL), USER_IN_ORGANIZATION_X_ADMIN_ROLE_ID, ORGANIZATION_X_ADMIN.roleName()));
-        when(jwtDecoder.decode(USER_IN_ORGANIZATION_Y_ADMIN_ROLE_JWT_TOKEN)).thenReturn(buildJwtTokenWithEmailCustomClaim(USER_IN_ORGANIZATION_Y_ADMIN_ROLE_JWT_TOKEN, of(USER_IN_ORGANIZATION_Y_ADMIN_ROLE_EMAIL), USER_IN_ORGANIZATION_Y_ADMIN_ROLE_ID, ORGANIZATION_Y_ADMIN.roleName()));
+        when(jwtDecoder.decode(USER_IN_ORGANIZATION_X_ADMIN_ROLE_JWT_TOKEN)).thenReturn(buildJwtTokenWithEmailCustomClaim(USER_IN_ORGANIZATION_X_ADMIN_ROLE_JWT_TOKEN, of(USER_IN_ORGANIZATION_X_ADMIN_ROLE_EMAIL), USER_IN_ORGANIZATION_X_ADMIN_ROLE_ID ));
```

Comment:
Maybe better to rename the "user in org admin role" constants to "organization XXX admin" since there are no roles involved anymore?

Reply:
Done